scripts/igraph_impl.py

```diff
@@ -0,0 +1,51 @@
+def overall_reciprocity(G):
+    return G.reciprocity()
+
+
+def pagerank(
+    G,
+    alpha=0.85,
+    personalization=None,
+    max_iter=100,
+    tol=1e-06,
+    nstart=None,
+    weight="weight",
+    dangling=None,
+    *,
+    vertices=None,
+    directed=True,
+    arpack_options=None,
+    implementation="prpack",
+):
+    if personalization is not None:
+        raise NotImplementedError
+    if nstart is not None:
+        raise NotImplementedError
+    if dangling is not None:
+        raise NotImplementedError
```

Comment:
Just confirming that the behaviour is the same as with `dangling=None` in NetworkX.